blues/version.py

```diff
@@ -2,6 +2,6 @@
 # This file is automatically generated by setup.py
 short_version = '0.0.1'
 version = '0.0.1.dev2'
-full_version = '0.0.1.dev2-0d9bf4c'
-git_revision = '0d9bf4cc2ca5931ac122c6375ae1af5f73a31df8'
+full_version = '0.0.1.dev2-571a2d7'
+git_revision = '571a2d7dce71d41505e8f9c0176645e6cce53726'
```

Comment:
I've not seen this before. Do these get added automatically via some tool, or are you doing this manually?

(Note that I've seen people using `versioneer` for automatic versioning on GitHub, and it's on my list to look into, though I've not done so yet. Just FYI, in case it is helpful.)